jped-cli/pom.xml

```diff
@@ -27,20 +27,20 @@
 			<version>1.3.1</version>
 		</dependency>
 		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<version>4.11</version>
-			<scope>test</scope>
-		</dependency>
-		<dependency>
 			<groupId>com.github.samtools</groupId>
 			<artifactId>htsjdk</artifactId>
-			<version>1.128</version>
```

Comment:
Indentation looks broken.

Could you verify that the indentation is consistent in all of your patches?